examples/src/main/resources/application.conf

```diff
@@ -0,0 +1,2 @@
+# Used by the Play example
+play.http.secret.key = "changeme"
```

Comment:
Not really, line removed. However, we need to maintain the `application.conf` because it seems that play loads it by default. Without it, I'm getting the next error: 
```[error] Configuration error: Configuration error[application: application.conf: java.io.IOException: resource not found on classpath: application.conf, application.json: java.io.IOException: resource not found on classpath: application.json, application.properties: java.io.IOException: resource not found on classpath: application.properties]```